readme.md

```diff
@@ -0,0 +1,30 @@
+Earl is a Discord bot designed to try to be your friend, but also to poke your buttons and be a bit provacative.
+He responds and reacts to certain phrases, words, and the like, to try to help uplift and motivate your community.
+Earl is a project developed by Breadpudding (Alex) and his friends out of their passion to create.
+
+# Who is Earl?
+Earl is one of a few characters from Microsoft Windows XP, specifically the [Microsoft Agents](https://tmafe.com/download-center-software).
+As stated in the [Fandom page](https://the-microsoft-agent.fandom.com/wiki/Earl) Earl is a yellow alien riding a purple and white surfboard.
+
+# What does this bot do?
+Earl uses RegEx string matching to recognize respond or react to phrases and words.
+Earl runs idley and waits for a matching pattern from his patterns.yml YAML file to appear, and then responds with the corrosponding response or maybe even reaction emoji.
+
+# Known issues:
+- Earl is seemingly unable to respond to custom emojis outside of the same server that the emoji identifier is from.
```

Comment:
We should probably just open an issue for this. No need to put this in the README